prjxray/mapping.yaml

```diff
@@ -0,0 +1,162 @@
+# part number to device, package and speed grade mapping
```

Comment:
The mapping file should be part of the database, as after this is merged,  as #1476 will auto-generate this file.

More detailed thoughts:
 - We should have a mapping file per family
 - The "pins" map probably belongs somewhere else.  The pins stuff is more likely to be hand-crafted, where as the other parts of the mapping can and eventually should be auto generated.

Reply:
Ah yes, moving the mapping file into the family directories will solve some issues I had.
I will undo the pins. My idea was to store these infos in same machine-readable files to automate the Makefiles a little bit. But since there are way more informations in these Makefiles, this might be done later or not.

Reply:
> Ah yes, moving the mapping file into the family directories will solve some issues I had.
> I will undo the pins. My idea was to store these infos in same machine-readable files to automate the Makefiles a little bit. But since there are way more informations in these Makefiles, this might be done later or not.

I'm totally fine with the pins change, but it should be seperate from the fabric stuff.

Reply:
Not sure if "resources.yaml" are the correct name for these information?!